src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/JointVelocityStateDynamics.h

```diff
@@ -0,0 +1,119 @@
+/**
+ * @file JointVelocityStateDynamics.h
+ * @authors Ines Sorrentino
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_ESTIMATORS_JOINT_VELOCITY_STATE_DYNAMICS_H
+#define BIPEDAL_LOCOMOTION_ESTIMATORS_JOINT_VELOCITY_STATE_DYNAMICS_H
+
+#include <memory>
+#include <map>
+
+#include <BipedalLocomotion/Math/Wrench.h>
+#include <BipedalLocomotion/RobotDynamicsEstimator/Dynamics.h>
+#include <BipedalLocomotion/RobotDynamicsEstimator/SubModel.h>
+#include <BipedalLocomotion/RobotDynamicsEstimator/SubModelKinDynWrapper.h>
+
+namespace BipedalLocomotion
+{
+namespace Estimators
+{
+namespace RobotDynamicsEstimator
+{
+
+/**
+ * The JointVelocityDynamics class is a concrete implementation of the Dynamics.
+ * Please use this element if you want to use the robot dynaic model to update the joint dynamics.
+ * The JointVelocityDynamics represents the following equation in the continuous time:
+ * \f[
+ * \ddot{s} = H^{-1} [\tau_{m} - \tau_{F} + (\sum J^T_{FT} f_{FT}) +
+ * + (\sum J^T_{ext} f_{ext})   - F^T {}^B \dot v  - h ]
+ * \f]
+ */
+
+class JointVelocityStateDynamics : public Dynamics
+{
+    double m_dT; /**< Sampling time. */
```

Comment:
From https://github.com/ami-iit/bipedal-locomotion-framework/pull/630 we started using `std::chrono::nanoseconds` for handling the time. Could you please use it also here?